python/hopsworks/__init__.py

```diff
@@ -241,14 +241,6 @@ def _prompt_project(valid_connection, project):
 
 
 def logout():
-    global _connected_project
-    if _connected_project is not None:
```

Comment:
Would it hurt to close it here too?

Reply:
It will actually close it when doing _saas_connection.close() on the lines below, this is the connection formed when doing hopsworks.login(). This code was written under the assumption that hopsworks.login() should only be used for app hence the naming. I'll rename _saas_connection to something else.

Reply:
Pushed the rename